src/cli.js

```diff
@@ -64,6 +64,6 @@ function run(str, opts, isWatch) {
 						'\n'
 				);
 			}
-			process.exit(err.code || 1);
+			process.exit(typeof err.code == 'number' ? err.code : 1);
```

Comment:
Caught this bug while running these tests - some NodeJS errors have the `.code` property set to a string, which if you pass here will cause the process to exit cleanly